src/test/resources/unrc/dose/Bono.feature

```diff
@@ -0,0 +1,91 @@
+
+Feature: test based challenge equals
+     Scenario Outline: User solves an unsolved test based challenge.
+Given that user <user> is already logged on 
+And the user wants to solve the test based challenge
+And the user has already solve this challenge before with a score of 12 points
+And the text of this challenge is
+'''
+	Public static boolean equals (int a, int b){
```

Comment:
java reserved words should be written in lowercase `public` instead of `Public`, code format should also take into account that there should be no spaces between a method's name and its arguments; there should be a space between the arguments closing parenthesis and the opening curly bracket.

The code of the method have the same issues: `If` instead of `if`; `else` statement should start after the closing curly bracket of the `if` statement; there should be a space between `else` and `{`.

It's preferable to have `return <condition>` instead of `if (condition) return true; else return false;`